openpype/hosts/maya/plugins/publish/extract_camera_mayaScene.py

```diff
@@ -219,3 +235,28 @@ def process(self, instance):
 
         self.log.info("Extracted instance '{0}' to: {1}".format(
             instance.name, path))
+
+    @classmethod
+    def get_attribute_defs(cls):
+        defs = super(ExtractCameraMayaScene, cls).get_attribute_defs()
+
+        defs.extend([
+            BoolDef("keep_input_connections",
+                    label="Keep Input Connections",
+                    tooltip="Preserving input connections will allow Image Planes to stay connected on publish",  # noqa
+                    default=cls.keep_input_connections),
+
+        ])
+
+        return defs
+
+    def ensure_image_plane_attachment(self, camera):
+        image_planes = cmds.listConnections(camera, type="imagePlane")
+        if not image_planes:
+            return
+
+        image_planes = [x.split("->")[1] for x in image_planes]
+        for image_plane in image_planes:
+            self.log.debug("reattaching {}".format(image_plane))
+            cmds.imagePlane(image_plane, edit=True, detach=True)
+            cmds.imagePlane(image_plane, edit=True, camera=camera)
```

Comment:
This should be a context manager so it's reverted after the export/baking because otherwise the user now is left with a scene without image plane attached after the export (and thus the next export also suddenly doesn't have an image plane). Right? Or does it attach the image plane to BOTH cameras?

Reply:
Think we can just wrap the code in the `delete_after` context for ease.

Reply:
Baked camera is deleted after export, Image Plane is kept on original camera